index.html

```diff
@@ -1913,6 +1919,67 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    When accessing data or requesting permission, [=sites=] (and other [=actors=]) should provide
+    [=people=] with relevant information about its use and [=user agents=] should help present and
+    consume that information.
```

Comment:
Browsers have historically shied away from showing a site's explanation of its permission request alongside the permission request, mostly because we can't guarantee that the site is telling the truth and don't want to endorse their claim. I'm worried that this principle is going to argue against that practice.

Reply:
Help in presenting and consuming information doesn't require any particular UI about permissions requests. I think we're agreed that this wouldn't prohibit any current browser vendor preference about UI.

(Although, also, yes, browsers should change their practices to improve visibility of the site's explanation when permissions are being requested, as the current model of providing no context or explanation whatsoever has proved extremely ineffective.)